experimental/bits/simd_fixed_size.h

```diff
@@ -395,6 +395,15 @@ template <typename _Tp, typename _Abi0, typename... _Abis>
 	  return second.template _M_simd_at<_Np - 1>();
       }
 
+    template <size_t _Offset>
+      _GLIBCXX_SIMD_INTRINSIC constexpr auto _M_tuple_at() const
+      {
+	if constexpr (_Offset == 0)
+	  return first;
+	else
+	  return second.template _M_tuple_at<_Offset - simd_size_v<_Tp, _Abi0>>();
+      }
+
```

Comment:
From the name I expected it to do the same as `_M_at`. Maybe call it `_M_tuple_with_element<N>`. However, I think you can't use this function anyway; see below.
Also, what if `_Offset > 0` but `_Offset < simd_size_v<_Tp, _Abi0>`?